src/main/java/org/jboss/set/overview/ejb/Aider.java

```diff
@@ -322,13 +348,17 @@ public void updatePullRequestData() {
     @Schedule(hour = "1,3,5,7,9,11,13,15,17,19,21,23")
     public void updatePayloadData() {
         if (devProfile) return;
```

Comment:
If we start it with devProfile by `-Dprbz-dev=true`, no update will happen. Can you just update the status to "Complete" and log a message before return ? Otherwise, it stops at "SCHEDULED".

Reply:
Thanks, missed that. I'm not 100% happy with the Status idea tbh. Wondering if it'd be better to have a status-per-refresh-request rather then global one. This way you could call /rest/api/refresh/{stream}/{release} multiple times, get a job id per each refresh and could query each one separately. Maybe an improvement further down the line.